course_flow/context_processors.py

```diff
@@ -5,13 +5,17 @@
 
 
 def update_notifications(request):
-    updates = UpdateNotification.objects.order_by("-created_on")
-    if updates.count() > 0:
-        last_update = updates.first()
-        last_update_serialized = (
-            JSONRenderer()
-            .render(UpdateNotificationSerializer(last_update).data)
-            .decode("utf-8")
-        )
-        return {"update_notifications": last_update_serialized}
+    try:
+        if "course-flow" in request.path:
```

Comment:
This is a little bit brittle as the django philosophy is for changes to urls.py to be without side effect.  I was hoping we could use some attribute of the request other than the path... not as direct as I hoped but this was discussed here [https://code.djangoproject.com/ticket/29341] with a solution.  Fine if you want to just leave it.

Reply:
Yeah I didn't like this either, but was at a bit of a loss as to how to tell otherwise if its a course-flow url. I had looked up my options and most answers on stackexchange seemed to be saying it was not possible to tell from the request, which is odd because this solution you found seems to contradict that. Regardless this is a much nicer solution. Do you think I can directly use `if "course_flow" in request.resolver_match.namespace` or is there a clean way of getting the namespace for your app from settings or something?

Reply:
I think that should work.  Worth testing anyway.